build.mill

```diff
@@ -14,13 +14,14 @@ import coursier.maven.MavenRepository
 
 object v {
   val scala    = "2.13.16"
```

Comment:
Do we still check the format of build file?

Reply:
No, the format check requires ScalafmtModule, so we have to format them manually.

Reply:
Ok please add CI check for format in the next PR